### PR TITLE
Add info about DGMM 1st

### LINKED ISSUE #60: Add new block for DGMM history
It could be nice to add a block like previous edition, including best paper price ?

### PR DESCRIPTION
**Pull request by PerretB**
fixes #60

### CODE REVIEW

File: menu.html [hunk 1]
```diff
@@ -29,21 +29,27 @@
...
 <!---  DEBUT Menu Events & Positions  --->
-	  <li class="navbar-brand dropdown active">
+
+	 <!-- Positions removed
+		<li class="navbar-brand dropdown active">
 			<a class="nav-link dropdown-toggle item" data-target="events.html"  href="events.html"   
 				   id="dropdown01" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Events and Positions</a>
 			<div class="dropdown-menu" aria-labelledby="dropdown01">
 			  <a class="dropdown-item" href="events.html">Events</a>
 			  <a class="dropdown-item" href="announce.html">Annoucements for positions</a>
```
Comment (kerautret) [inline menu.html:46]: You remove the announce position page ? Less and less use perhaps not needed outside to see archive.

File: events.html [hunk 2]
```diff
@@ -116,101 +116,103 @@ <h3>TC18 newsletters</h3>
...
-        <div dgciReview="dgciReview.html" > </div>
-        <script>includeHTML("dgciReview");</script>
+
+        <h3>DGMM: International Conference on Discrete Geometry and Mathematical Morphology</h3>
+        <a  class="shiftedAnchor" name="dgmm"></a>
+            
+            <p>The conference on Discrete Geometry and Mathematical Morphology (DGMM) is a joint event between the 
+                two main conference series of IAPR TC18, the International Conference on Discrete Geometry for 
+                Computer Imagery (DGCI) and the International Symposium on Mathematical Morphology (ISMM). 
+                The first DGMM edition happened in Uppsala, Sweden, in 2021, 5 years after the extension of the scope
```
Comment (kerautret) [inline events.html:126]:
```suggestion
                The first DGMM edition happened in Uppsala, Sweden, in May 2021, 5 years after the extension of the scope 
```

File: dgmmReview.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                            <li> <a href="https://people.cmm.minesparis.psl.eu/users/angulo/"> Jesús Angulo</a> (CMM-MINES ParisTech, France):<br> <a href="https://arxiv.org/pdf/2105.01339.pdf">"Some open questions on morphological operators and representations in the deep learning era"</a></li>
+                            <li> <a href="https://katalog.uu.se/profile/?id=N5-824"> Cecilia Holmgren</a> (Uppsala University, Sweden):<br> <a href="https://link.springer.com/chapter/10.1007/978-3-030-76657-3_2">"A unifying model for many important random trees of logarithmic height"</a></li>
+                            
+                            </ul>
+                        </li>
+                        <li> Best Student Paper award: Rémi Decelle from Université de Lorraine for the work entitled <a href="https://hal.archives-ouvertes.fr/hal-03144152/file/paper_47.pdf">"Digital Straight Segment Filter for Geometric Description"</a> (with Phuc Ngo, Isabelle Debled-Rennesson, Frédéric Mothe and Fleur Longuetaud)</li>
```
Comment (kerautret) [inline dgmmReview.html:26]: If I am not wrong another was mentioned  (as "distinction"?) : Combining Deep Learning and Mathematical Morphology for Historical Map Segmentation Yizi Chen, Edwin Carlinet, Joseph Chazalon, Clément Mallet, Bertrand Duménieu and Julien Perret

**Review by kerautret — APPROVED** [review]
Looks fine even if I not tested, but you could check after merge ;)

File: menu.html (hunk 1 above)
Comment (PerretB) [reply menu.html:46]: Yes, the only open position was probably largely outdated (2 years olds). We could think about reestablishing the page if we find a way to make people use it

File: dgmmReview.html (hunk 3 above)
Comment (PerretB) [reply dgmmReview.html:26]: you're right, I will add it once they update the website to see exactly how they called it

**Pull request merged**